### ISSUE TITLE
good mAP but low recall

### ISSUE THREAD

**Issue by qazalkz** [tensorflow/models#11125]
hello guys. i trained my model for several times with efficiendnet3 , so changed hyperparameters to improve my mAP. but the recall doesn't change: 40~60
how can i improve it. my dataset contains 8300 images.
DONE (t=0.33s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.361
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.919
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.132
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.276
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.392
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.472
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.415
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.478
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.487
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.446
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.504
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.517
INFO:tensorflow:Eval metrics at step 92000

also based on which recall i can calculate f1 score? my main threshold is 0.5.

INFO:tensorflow:	+ Loss/localization_loss: 0.007066
I1205 14:11:19.652778 140292449477888 model_lib_v2.py:1018] 	+ Loss/localization_loss: 0.007066
INFO:tensorflow:	+ Loss/classification_loss: 0.300997
I1205 14:11:19.653136 140292449477888 model_lib_v2.py:1018] 	+ Loss/classification_loss: 0.300997
INFO:tensorflow:	+ Loss/regularization_loss: 0.043959
I1205 14:11:19.653499 140292449477888 model_lib_v2.py:1018] 	+ Loss/regularization_loss: 0.043959
INFO:tensorflow:	+ Loss/total_loss: 0.352023
I1205 14:11:19.653853 140292449477888 model_lib_v2.py:1018] 	+ Loss/total_loss: 0.352023

**Comment by laxmareddyp** [comment]
Hi @qazalkz ,

Sorry for delay in response and To assist you more efficiently, could you please share a code sample or a link to a Google Colab notebook that reproduces the issue you're experiencing? Additionally, it would be helpful to know whether you are using the official TensorFlow Model Garden repository or a research-based one.

Thanks.
